frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card-body.tsx

```diff
@@ -75,32 +57,31 @@ export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({ isLoading, m
       </EmptyState>
     );
   }
+  const totalHumanized = humanizeBinaryBytesWithoutB(totalUsage, null, totalUsage ? null : 'Mi');
```

Comment:
Let use humanizeBinaryBytes where ever we are using it now.
```suggestion
  const totalHumanized = humanizeBinaryBytes(totalUsage, null, totalUsage ? null : 'Mi');
```

Reply:
Just an FYI, 
`humanizeBinaryBytes` are backed by `binaryBytes` data structure and that looks something like this,
```javascript
  binaryBytes: {
    units: ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'],
    space: true,
    divisor: 1024,
  },
```
The `units` array don't have 'EiB' in it (ideally it should have, in order to be consistent with other data structures)

Reply:
@aruniiird Thanks, I think you can send a separate PR to fix this.

Reply:
@cloudbehl , made the changes... Some screen shot,

![cu_card_with_humanizeBinaryBytes](https://user-images.githubusercontent.com/895234/62773614-365c0c00-bac0-11e9-82e9-fe0995c744d7.png)

PS : For large values (mostly corner cases) we may see an odd behavior with the total chart-title (text in the middle).

Reply:
hm, you're right about those large values. How about moving units below the number ? @yuvalgalanti

Reply:
`font-size: 1.2rem;` seems to work well up to `88.88 GiB` and seems to look better in general, maybe we could do that?

I know we've also talked about rounding to the nearest 1 decimal place ([see here](https://docs.google.com/a/redhat.com/document/d/1oT2n1S7m9wR9Fg8MmQogvLXyRiBgUS_yNFhE0bszMVw/edit?disco=AAAADPPwrHE)) but that's not confirmed yet.

Reply:
> hm, you're right about those large values. How about moving units below the number ? @yuvalgalanti

Already tried that with my limited knowledge, but was unsuccessful there. The above value (let say `2.18 GiB` from the sample) is passed to one of the `ChartDonut`'s property (`title`), so I am not sure how to make it multi-line (or do any other format changes) etc.
@afreen23 , @cloudbehl , @rawagner , @yuvalgalanti  thoughts???

Reply:
@aruniiird Would my `font-size` suggestion above work using the React component's `style` prop?

The backup plan might be to move the unit label into the `subTitle` prop and make more styling tweaks, but I think just adjusting the font size via the `style` prop would be simpler/better.

Reply:
> @aruniiird Would my font-size suggestion above work using the React component's style prop?

Yes, it works perfectly fine (two thumbs up)...
With @andybraren 's suggestion,

![cb_with_big_total](https://user-images.githubusercontent.com/895234/62815844-3994dd80-bb3c-11e9-879d-ae0ae9099612.png)

A few changes which would help here are,

a. `comma` being removed from `humanizedData.value` (@rawagner 's PR)...
b. if `EiB` is added to `humanizeBinaryBytes` (please see above comments and a PR is already up)
c. if we could use the `WithoutB` formats of `humanize` functions

Suggestions ???

Reply:
@andybraren , on further test, the style-fontSize combination is messing up the `tooltip`. So I am not sending a PR with those changes.

Reply:
So close! Really weird that the tooltip would be affected by changing the fontSize.

This is a long shot, but would `1.2em` without the `r` work?

I think we'd prefer to keep `comma` and use `XiB` (if #2296 is approved, I could use help there). `b` sounds good though.

Reply:
I tried something like this, [ just to make sure everything written as expected ]...

```typescript
return (
  <div className="nb-capacity-card__chart-body">
    <div className="noobaa-capacity-card__item">
      <ChartDonut
        width={275}
        height={150}
        labelPosition="centroid"
        legendPosition="right"
        legendData={metricLegend}
        legendOrientation="vertical"
        data={metricData}
        labels={(datum) => `${datum.x}: ${humanizePercentage(totalUsage ? datum.y : 0).string}`}
        subTitlePosition="bottom"
        themeColor={ChartThemeColor.multi}
        themeVariant={ChartThemeVariant.light}
        title={totalHumanized.string}
        style={{ labels: { fontSize: '1em' } }}
      />
    </div>
  </div>
);
```

![image](https://user-images.githubusercontent.com/895234/62835395-b2e41b80-bc75-11e9-9df3-28189ee249a0.png)

@andybraren , any idea...

Reply:
> Here's what I see when I test something close to 1,024, at which point the unit label should increase to the next level. Seems fine if we can somehow get the fontSize to work without affecting the tooltip.

That will work even without the `style` change (it will be a bit bulkier and to the brim)...

![image](https://user-images.githubusercontent.com/895234/62835661-a6ad8d80-bc78-11e9-81b7-96b5a5400834.png)

Reply:
> style={{ labels: { fontSize: '1.2em' } }}

My limited knowledge of React is failing me here 😞. Can the 1.2em be applied to something other than `labels` (plural)? Maybe `title`?

I'm not really sure why the black tooltip would be small in your earlier screenshot. If your most recent screenshot is the closest we can get then I'm happy enough for now 🙂.